src/mango.js

```diff
@@ -66,7 +66,21 @@ function defineIndexV3 (cozy, doctype, fields) {
   let path = createPath(cozy, false, doctype, '_index')
   let indexDefinition = {'index': {fields}}
   return cozyFetchJSON(cozy, 'POST', path, indexDefinition)
-    .then((response) => ({ doctype: doctype, type: 'mango', name: response.id, fields: fields }))
+    .then((response) => {
+      const indexResult = { doctype: doctype, type: 'mango', name: response.id, fields }
+      const opts = getV3Options(indexResult, {'selector': {_id: {'$gt': null}}})
+      let path = createPath(cozy, false, indexResult.doctype, '_find')
+      return cozyFetchJSON(cozy, 'POST', path, opts)
+      .then(() => {
+        return indexResult
+      })
+      .catch(() => { // one retry
+        return cozyFetchJSON(cozy, 'POST', path, opts)
```

Comment:
I think you should wait a bit before retrying

Reply:
Indeed, you're right 👍